src/components/DocExplorer.js

```diff
@@ -496,7 +496,7 @@ class TypeDoc extends React.Component {
                 {': '}
                 <TypeLink type={field.type} onClick={onClickType} />
                 {
-                  field.isDeprecated &&
+                  field.deprecationReason &&
```

Comment:
Hmm, what's happening here?

It's entirely possible that a field is deprecated but also fails to provide a reason.

The previous code looks more correct, so we may need to look elsewhere for the source of the issue.